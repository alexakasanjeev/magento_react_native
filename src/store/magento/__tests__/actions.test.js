import {
  initializeApp,
  changeCurrency,
} from '../actions';

describe('magento actions', () => {
  test('initializeApp()', () => {
    // Exercise
    const result = initializeApp();

    // Verify
    expect(result).toMatchSnapshot();
  });

  test('changeCurrency()', () => {
    // Setup
    const currencyCode = 'USD';
    const currencySymbol = '$';
    const currencyExchangeRate = 1;

    // Exercise
    const result = changeCurrency(currencyCode, currencySymbol, currencyExchangeRate);

    // Verify
    expect(result).toMatchSnapshot();
  });
});
