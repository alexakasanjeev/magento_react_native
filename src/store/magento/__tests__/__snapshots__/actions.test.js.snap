// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`magento actions changeCurrency() 1`] = `
Object {
  "payload": Object {
    "currencyCode": "USD",
    "currencyExchangeRate": 1,
    "currencySymbol": "$",
  },
  "type": "CHANGE_CURRENCY",
}
`;

exports[`magento actions initializeApp() 1`] = `
Object {
  "type": "MAGENTO_INIT_APP_REQUEST",
}
`;
